Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8022,33 +8013,31 @@ def total(self):
 
     @property
     def limit(self):
-        return self._next_limit
+        return self._limit
 
     @limit.setter
     def limit(self, value):
-        self._next_limit = self._original_limit = value
+        self._limit = value
 
-    def _is_search_done(self):
+    def is_search_done(self):
         """
         Checks one of these conditions:
         1. self.limit is set, and it's updated to be less or equal to zero
```

Comment:
```suggestion
        1. self.limit is set, and it's updated to be less or equal to zero - return True
```

Mention the return value in each case.